File: Lab2/app/src/main/java/com/example/firstjavaapp/models/Review.java
package com.example.firstjavaapp.models;


import java.util.ArrayList;

public class Review {
        public ArrayList<DocReview> docs;
        public int total;
        public int limit;
        public int page;
        public int pages;

        @Override
        public String toString(){
                return String.valueOf(this.docs.get(0).review) + " , всего отзывов: " + this.total;
        }
}

File: Lab_2_NEW/app/src/main/java/com/example/lab_2_new/models/Review.kt
package com.example.lab_2_new.models

class Review {
    var docs: ArrayList<DocReview>? = null
    var total = 0
    var limit = 0
    var page = 0
    var pages = 0

    override fun toString(): String {
        return java.lang.String.valueOf(docs!![0].review).toString() + " , всего отзывов: " + total
    }
}